openff/toolkit/topology/molecule.py

```diff
@@ -3224,6 +3224,7 @@ def generate_conformers(
                 n_conformers=n_conformers,
                 rms_cutoff=rms_cutoff,
                 clear_existing=clear_existing,
+                raise_exception_types=[],
```

Comment:
(blocking) This should get a "Behavior changed" entry in the releasehistory, since folks may have implemented logic that expects conf gen failures from a single ToolkitWrapper in a larger ToolkitRegistry to be fatal in this method.

Reply:
42ed44b - wording could be smithed but I'll leave that for the pre-release cleanup